extensions/yaml/package.json

```diff
@@ -13,6 +13,18 @@
   },
   "contributes": {
     "languages": [
+      {
+        "id": "compose",
```

Comment:
Would it make sense to disambiguate it as `"docker-compose"`?

Reply:
Seems reasonable to me. I can change it.